ViAn/Filehandler/project.h

```diff
@@ -15,7 +15,7 @@ typedef int ID;
  * Importand for seperation of different types of data.
  */
 struct ProjectStream{
-    std::stringstream projFile;
+    std::stringstream projfile;
```

Comment:
no, this is a middle-hand for reading seperate projectfiles. this one for reading project infofile